src/cpp/r/RSexp.cpp

```diff
@@ -97,6 +99,21 @@ bool fillSetString(SEXP object, std::set<std::string>* pSet)
    return true;
 }
 
+std::vector<std::string> getLoadedNamespaces()
+{
+   std::vector<std::string> result;
+   
+   r::exec::RFunction loadedNamespaces("loadedNamespaces");
+   Protect protect;
+   SEXP resultSEXP;
+   
+   Error error = loadedNamespaces.call(&resultSEXP, &protect);
+   IF_ERROR(error, return result);
+   
+   fillVectorString(resultSEXP, &result);
```

Comment:
I think you can just pass &result to call and it will automatically do the conversion with no intermediate SEXP to manage (I could be missing something about this case though that makes that impractical)